docs/references/configuration.md

```diff
@@ -0,0 +1,192 @@
+---
+description: Parameter reference
+---
+
+# Configuration
+
+When installing Opstrace, one has to provide a corresponding configuration document in YAML format.
```

Comment:
> When installing Opstrace

I know this is old content, buuut WDYT about 

"When creating a new Opstrace instance, ..."

Reply:
This is a longer conversation, but no, I don't think that we just s/cluster/instance/.  The word "instance" is not necessary in many cases.